src/bci_build/package/__init__.py

```diff
@@ -249,8 +249,9 @@ def has_container_suseconnect(self) -> bool:
     def eula_package_names(self) -> tuple[str, ...]:
         if self.is_sle15:
             return ("skelcd-EULA-bci",)
-        # if self.is_slcc:
-        #     return (f"skelcd-EULA-{str(self.value).lower()}",)
+        # TODO: switch to skelcd-EULA-bci when SLES 16 is released
```

Comment:
Do we have a ticket/bug/issue somewhere to not forget this TODO?

Reply:
BCI-tests will tell us that (we have a test for the eula iirc). but I created https://github.com/SUSE/BCI-dockerfile-generator/issues/1666